InnoClinic.OfficesAPI/API/Program.cs

```diff
@@ -1,10 +1,22 @@
-﻿using BLL.Extensions;
+﻿using API.Middleware;
+using BLL.Extensions;
+using BLL.Validators;
+using FluentValidation;
+using FluentValidation.AspNetCore;
+using Microsoft.AspNetCore.Http.Features;
 
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddFluentValidationAutoValidation()
+    .AddFluentValidationClientsideAdapters()
+    .AddValidatorsFromAssemblyContaining<OfficeForCreationDtoValidator>();
```

Comment:
Are you sure it should be the first registered service?
Maybe better to move it to a separate IOC